keep-ui/app/alerts/alert-associate-incident-modal.tsx

```diff
@@ -61,7 +62,7 @@ const AlertAssociateIncidentModal = ({
       onClose={handleClose}
       title="Choose Incident"
       className="w-[600px]"
-    >
+    >{console.log(incidents)}
```

Comment:
Please tell me, is this information used for debugging? If so, I think it should be deleted.